### PR TITLE
docs(0.13.x/admin-api) document /upstream/:id/health endpoint

### PR DESCRIPTION
**Pull request by hishamhm**

### CODE REVIEW

File: app/docs/0.13.x/admin-api.md [hunk 1]
```diff
@@ -2253,6 +2253,73 @@ HTTP 204 No Content
...
+
+The `data` field of the response contains an array of target objects.
+The health for each target is returned in its `health` field:
+
+* If a target fails to be activated in the ring balancer due to DNS issues,
+  its status displays as `DNS_ERROR`.
+* When [health checks][healthchecks] are not enabled in the upstream
+  configuration, the health status for active targets is displayed as
+  `HEALTHCHECKS_OFF`.
+* When health checks are enabled and the target is determined to be healthy,
```
Comment (coopr) [inline app/docs/0.13.x/admin-api.md]: Also when it is manually marked as healthy?

File: app/docs/0.13.x/admin-api.md [hunk 2]
```diff
@@ -2253,6 +2253,73 @@ HTTP 204 No Content
...
+  its status is displayed as `HEALTHY`. This means that this target is
+  currently included in this Upstream's load balancer ring.
+* When a target has been disabled by either active or passive health checks
+  (circuit breakers), its status is displayed as `UNHEALTHY`. The load
```
Comment (coopr) [inline app/docs/0.13.x/admin-api.md]: Also when manually marked as unhealthy?

File: app/docs/0.13.x/admin-api.md (hunk 1 above)
Comment (hishamhm) [reply app/docs/0.13.x/admin-api.md]: Note the clever use of passive voice there: I didn't say who determined the target to be healthy. ;) But point taken, will fix!

File: app/docs/0.13.x/admin-api.md (hunk 2 above)
Comment (hishamhm) [reply app/docs/0.13.x/admin-api.md]: For this one I have no excuse. :)

File: app/docs/0.13.x/admin-api.md [hunk 3]
```diff
@@ -2253,6 +2253,75 @@ HTTP 204 No Content
 
 ---
 
+### Show upstream health for node
+
+Displays the health status for all targets of a given upstream, according to
```
Comment (thibaultcha) [inline app/docs/0.13.x/admin-api.md]: could we capitalize "targets" and "upstream" here and in other occurrences? We try to capitalize Kong entities through-ought the docs and I think we should stay consistent

Comment (coopr) [reply app/docs/0.13.x/admin-api.md]: Yes please, good catch @thibaultcha

**Comment by hishamhm** [comment]
Updated with uppercase entities.

**Comment by thibaultcha** [comment]
ditto

**Pull request closed without merging**